pkg/cmd/openshift-tests/monitor/run/run_monitor_command.go

```diff
@@ -186,7 +186,7 @@ func (o *RunMonitorOptions) Run() error {
 
 	fmt.Fprintf(o.Out, "Monitor shutting down, this may take up to five minutes...\n")
 
-	cleanupContext, cleanupCancel := context.WithTimeout(context.Background(), 5*time.Minute)
+	cleanupContext, cleanupCancel := context.WithTimeout(context.Background(), 15*time.Minute)
```

Comment:
I wound up bumping this to 20 minutes [master run_monitor_command](https://github.com/openshift/origin/blob/master/pkg/cmd/openshift-tests/monitor/run/run_monitor_command.go#L189)

Reply:
fixed